BACKERS.md

```diff
@@ -0,0 +1,87 @@
+# Sponsors & Backers
+
+Material-UI is an MIT-licensed open source project. It's an independent project with ongoing development made possible thanks to the support of these awesome backers. If you'd like to join them, please consider:
+- [Become a backer or sponsor on Patreon](https://www.patreon.com/oliviertassinari).
+- [Become a backer or sponsor on OpenCollective](https://opencollective.com/material-ui).
+
+Through contributions, donations, and sponsorship, you allow Material-UI to thrive.
+Your donations directly support office hours, continued enhancements, great documentation and learning materials!
```

Comment:
Your contributions, donations, and sponsorship allow us to build a sustainable organization. They directly support office hours, continued enhancements, great documentation and learning materials!